### DISCUSSION TITLE
install themes crashes and adds double wwwroot

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#694] [General] [answered]
we are workingo n a custom theme and when we install the nuget package it crashes.  one clue to the crash is that an extra wwwroot is added to the path.  I based my theme off the oqtane.theme.cerulean example and just changed a few styles and the name.

![image](https://user-images.githubusercontent.com/311063/90698487-aa81d180-e235-11ea-82a4-f13374ddb499.png)

my repo: https://github.com/DarkMatterOregon/oqtane.theme.newco

**Comment by sbwalker** [comment]
Which version of Oqtane are you using? There was an issue in 1.0.1 which was fixed in #670 and released in 1.0.2

**Reply by markdav-is** [reply]
Ohh, I have not been following along with versions!  I need to upgrade.  This will be my first time upgrading...

**Comment by sbwalker** [comment]
this might help: https://www.oqtane.org/Resources/Blog/PostId/543/upgrading-oqtane

**Reply by markdav-is** [reply]
interesting... I was looking at updating my fork from the upstream.  Maybe I'm being too nerdy about it.

**Reply by markdav-is** [reply]
@sbwalker  do you think I need to re-create my theme or is this bug fixed in the framework?

**Comment by sbwalker** [comment, marked as answer]
I believe it was a bug in the framework and it has been fixed
